CMakeLists.txt

```diff
@@ -19,9 +19,19 @@ set(SUPERBUILD_TOPLEVEL_PROJECT inner)
 #-----------------------------------------------------------------------------
 # Extension dependencies
 find_package(Slicer REQUIRED)
+# Skip implicit generation of launcher settings triggered by SlicerBlockAdditionalLauncherSettings
+# It was generating the CMake Warning "Dependent extension SlicerIGT cannot be found by CMake find_package() ..."
+set(Slicer_SKIP_SlicerBlockAdditionalLauncherSettings TRUE)
 include(${Slicer_USE_FILE})
 mark_as_superbuild(Slicer_DIR)
 
+# Explicitly include SlicerBlockAdditionalLauncherSettings after resetting EXTENSION_DEPENDS.
+# Since SlicerIGT is a runtime dependency, is it not needed for generating the launcher settings.
```

Comment:
Are SlicerIGT modules still going to be available when we start Slicer using the generated launcher?